inc/class-sitemaps.php

```diff
@@ -1598,3 +1528,101 @@ private function calculate_priority( $post ) {
 	}
 
 } /* End of class */
+
+/**
+ * Class WPSEO_Sitemap_Timezone
+ */
+class WPSEO_Sitemap_Timezone {
```

Comment:
Is there a reason for there being two classes in one file? As a best practice I think it is better to have 1 class per file, 1 file per class.

Furthermore I think a better name for this class would be `WPSEO_Timezone`. There is no sitemap specific code here.